src/base/system_parameter.c

```diff
@@ -391,6 +391,10 @@ static const char sysprm_ha_conf_file_name[] = "cubrid_ha.conf";
 
 #define PRM_NAME_HA_SQL_LOG_MAX_SIZE_IN_MB "ha_sql_log_max_size_in_mbytes"
 
+#define PRM_NAME_HA_SQL_LOG_PATH "ha_sql_log_path"
+
+#define PRM_NAME_HA_SQL_LOG_KEEP_COUNT "ha_sql_log_keep_count"
```

Comment:
```suggestion
#define PRM_NAME_HA_SQL_LOG_KEEP_COUNT "ha_sql_log_max_count"
```
NIT. "max" seems to me more intuitive and broadly used. at your choice.